scripts/output/single/ar6Climate.R

```diff
@@ -0,0 +1,152 @@
+library(madrat)
+library(remind2)
+library(quitte)
+library(piamInterfaces)
+library(yaml)
+library(lucode2)
+library(readxl) # GA: Wont be necessary after https://github.com/iiasa/climate-assessment/pull/43 goes into release
+
+############################# BASIC CONFIGURATION #############################
+gdx_name     <- "fulldata.gdx"             # name of the gdx
+cfg_name     <- "cfg.txt"                  # cfg file for getting file paths
+
+
+if(!exists("source_include")) {
+   # Define arguments that can be read from command line
+   outputdir <- "."
+   readArgs("outputdir", "gdx_name", "gdx_ref_name", "gdx_refpolicycost_name")
+}
+
+gdx           <- file.path(outputdir, gdx_name)
+cfg_path      <- file.path(outputdir, cfg_name)
+logfile       <- file.path(outputdir, "climate.log") # specific log for python steps
```

Comment:
Maybe I would prefer `log_climate.txt`, similar to `log_renv.txt`, `log_output.txt` etc.

Reply:
Done